systest/cluster/cluster.go

```diff
@@ -98,6 +98,9 @@ func New(cctx *testcontext.Context, opts ...Opt) *Cluster {
 	cluster.addFlag(genesis)
 	cluster.addFlag(TargetOutbound(defaultTargetOutbound(cctx.ClusterSize)))
 	cluster.addFlag(NetworkID(defaultNetID))
+	cluster.addFlag(CycleGap(10 * time.Second))
+	cluster.addFlag(PhaseShift(20 * time.Second))
+	cluster.addFlag(GracePeriod(10 * time.Second))
```

Comment:
this can't be 10s. next poet round starts execution at this time. code needs to stop waiting some time before the start, so that it has time to prepare challenge and publish it

5s doesn't work?